src/Redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,55 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const url = 'https://api.spacexdata.com/v4/rockets';
+
+const FETCH_ROCKETS = 'space-travelers-react-redux/rockets/fetchRockets';
+
+// Action creator
+export const fetchRockets = createAsyncThunk(FETCH_ROCKETS, async () => {
+  const response = await fetch(url);
+  const data = await response.json();
+  return data;
+});
```

Comment:
- Kindly, you are storing the whole API response object which is not necessary, Please map and store only used data to redux store to improve its performance as you did for rockets 🚀
![2023-04-15_04-47](https://user-images.githubusercontent.com/52010789/232177487-2d5b4084-d8d9-4c28-a386-511166af9fc2.png)

Reply:
Resolved